### ISSUE TITLE
Docker manifest missing

### ISSUE THREAD

**Issue by Virsacer** [AdguardTeam/AdGuardHome#5495]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to report a bug and not ask a question

### Operating system type

Linux, Other (please mention the version in the description)

### CPU architecture

64-bit ARM

### Installation

Docker

### Setup

On one machine

### AdGuard Home version

latest

#### What did you do?
    docker manifest inspect adguard/adguardhome | head -6
#### Expected result
    {
       "schemaVersion": 2,
       "mediaType": "application/vnd.docker.distribution.manifest.list.v2+json",
       "manifests": [
          {
             "mediaType": "application/vnd.docker.distribution.manifest.v2+json",

#### Actual result
    no such manifest: docker.io/adguard/adguardhome:latest
#### Additional information
Current docker image has no manifest.
My update-checker can not determine if local latest is the same as registrys latest.

Apparently there is an issue with `buildx` which might be the cause: https://github.com/docker/buildx/issues/1509

**Comment by ainar-g** [comment]
You are correct in that it's a Docker bug, see moby/moby#43126.  It seems like they've released a fix for that in v23.0.0, but it'll still be a while until these get into stable OS repos.

**Comment by ttyS0** [comment]
This also means the container can't be pulled by MikroTik systems.

**Comment by zanyarelectronic** [comment]
> This also means the container can't be pulled by MikroTik systems.

yes correct i cant pull it to mikrotik anymore and there is an error  (error response getting manifests: 404)

**Comment by ttyS0** [comment]
Problem still exists with v0.107.25.

```
 09:06:54 container,info,debug importing remote image: adguard/adguardhome, tag: v0.107.25
 09:06:54 container,info,debug error response getting manifests: 404
```

**Comment by ttyS0** [comment]
Problem still exists with v0.107.26

```
 07:56:18 container,info,debug importing remote image: adguard/adguardhome, tag: v0.107.26
 07:56:18 container,info,debug error response getting manifests: 404
```

**Comment by hhaniel** [comment]
I can pull with RHEL9/RHEL8 and then save image as tar then load it on RHEL7 - Can no longer pull direct with RHEL7 - last version I can still pull direct is v0.107.23

**Comment by ttyS0** [comment]
Problem still exists with v0.107.26

```
 09:25:11 container,info,debug importing remote image: adguard/adguardhome, tag: v0.107.27
 09:25:12 container,info,debug error response getting manifests: 404
```

**Comment by ttyS0** [comment]
Problem still exists with v0.107.28

```
 08:58:22 container,info,debug importing remote image: adguard/adguardhome, tag: v0.107.28
 08:58:22 container,info,debug error response getting manifests: 404
```

**Comment by Ziris85** [comment]
I know this is an old issue, but as it's still open, I figured I would just toss an update that this is still an issue as of v0.107.71. I was trying to set up a local docker image registry and it doesn't work with the `:latest` tag for adguard for the reason stated in this bug:
```
level=error msg="response completed with error" environment=development err.code="manifest unknown" err.detail="unknown tag=latest" err.message="manifest unknown" http.request.method=HEAD http.request.uri="/v2/adguard/adguardhome/manifests/latest?ns=docker.io"
```

**Comment by Virsacer** [comment]
I forgot about this issue...
So I dont know since when or what fixed it, but `docker manifest inspect adguard/adguardhome | head -6` output is as expected:

    {
       "schemaVersion": 2,
       "mediaType": "application/vnd.oci.image.index.v1+json",
       "manifests": [
          {
             "mediaType": "application/vnd.oci.image.manifest.v1+json",

**Comment by Ziris85** [comment]
Heh, yes, I take back my earlier comment. I was attempting to set up the Docker Distribution registry and was getting that error, so assumed it was the same thing. It just turned out I had it misconfigured. Everything looking good.

**Comment by ainar-g** [comment]
@Virsacer, thanks for letting us know!  Can the issue be closed now?
